<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputTime</class>
 <widget class="QWidget" name="InputTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>第1天</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>开始时间</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="timeEdit_2">
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>结束时间</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="timeEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>扣除休息时间/分钟</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
